getting-started/quickstart-guides/vuejs.md

```diff
@@ -365,8 +363,6 @@ export default {
 </script>
 ```
 
-> `authenticationEndpoint` should be implemented in your backend. The SDK makes an HTTP GET request to this endpoint and expects a JSON response with three fields i.e. `signature`, `token`, and `expire`. [Learn how to implement authenticationEndpoint](https://docs.imagekit.io/api-reference/upload-file-api/client-side-file-upload#how-to-implement-authenticationendpoint-endpoint) on your server.
-
 For this quickstart guide, we have provided a sample implementation of `http://localhost:3001/auth` in Node.js.
```

Comment:
This line alone doesn't make much sense. With above paragraph removed, the whole context is lost.

Reply:
Added description above